webapp/components/Editor/Editor.vue

```diff
@@ -1,6 +1,7 @@
 <template>
```

Comment:
Hey @Mogge, thank you for all of the really helpful feedback and the time for the suggestions. I would love to do a pairing session to work on the one small issue. I think it will help me understand the workflow for this project. Appreciate your time.